docs/we_made_this.md

```diff
@@ -35,6 +35,8 @@ A collection of articles explaining in detail how Lambda Powertools helps with a
 
 * [Part 6 - Configuration & Feature Flags](https://www.ranthebuilder.cloud/post/aws-lambda-cookbook-part-6-feature-flags-configuration-best-practices){:target="_blank"}
 
+* [Serverless API Idempotency with AWS Lambda Powertools and CDK](https://www.ranthebuilder.cloud/post/serverless-api-idempotency-with-aws-lambda-powertools-and-cdk){:target="_blank"}
```

Comment:
Hi @ran-isenberg great article! Can I just ask you to change all links to Powertools docs to use the prefix https://docs.powertools.aws.dev/lambda-python/latest/utilities/idempotency/?

It was not your fault, we've just been messing with the URLs, but that prefix should be final.

Let me know when you finish it and I'll be happy to merge and publish!

Reply:
I already have last week. Changed 13 articles. I have [ahref](https://ahrefs.com/) automation which found me all the broken links.